Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -1287,7 +1567,12 @@ public async Task issue_637()
 //using System.Text.Json;
 ");
 
-            events.Should().NotContainErrors();
+            events
+                .OfType<StandardErrorValueProduced>()
+                .Last()
+                .Value
+                .Should()
+                .Be("");
```

Comment:
Why this change? This doesn't look like the correct behavior.

Reply:
Hmm, it isn't ...